src/components/offcanvas/Offcanvas.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {default as RBOffcanvas} from 'react-bootstrap/Offcanvas';
+
+/**
+ * Create a toggleable hidden sidebar using the Modal component.
+ * Toggle the visibility with the `is_open` prop.
+ */
+const Offcanvas = props => {
+  const {
+    is_open,
+    setProps,
+    children,
+    class_name,
+    backdrop_class_name,
+    labelledby,
+    scrollable,
+    ...otherProps
+  } = props;
+
+  const onHide = () => {
+    if (setProps) {
+      setProps({is_open: !is_open});
+    }
+  };
+
+  // Left out:
+  // - container
+  // - autoFocus
+  // - enforceFocus
+  // - onEnter
+  // - onEntered
+  // - onEntering
+  // - onEscapeKeyDown
+  // - onExit
+  // - onExited
+  // - onExiting
+  // - onShow
+  // - restoreFocus
+  // - restoreFocusOptions
```

Comment:
Let's drop this comment before merging.

Reply:
Left here as we need to discuss which of these we might want to use.